app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-start">
+      <% if user_signed_in? %>
+        <%= link_to "Food", foods_path, class: "navbar-item has-text-link" %>
+        <%= link_to "Recipes", recipes_path, class: "navbar-item has-text-link" %>
+        <%= link_to "Public recipes", recipes_path, class: "navbar-item has-text-link" %>
+        <%= link_to "Shopping list", recipes_path, class: "navbar-item has-text-link" %>
```

Comment:
- [ ] The link for the shopping list, has to show the list of items that have been shopped or generated, however, I see the same items for public recipes and recipes as well.

![rails15](https://user-images.githubusercontent.com/55635977/142577130-67aa2b10-0bcd-43aa-952f-bd10d025dc2e.png)

- [ ] Has public recipes page listing all publicly available recipes from other users with their names and total prices.